bundle/Resources/config/routing.yml

```diff
@@ -0,0 +1,6 @@
+contentonthefly.location_controller.suggested:
+    path: /contentonthefly/locations/{content}
```

Comment:
can't this also be injected into ui as config instead of having to do a REST call for it?